### PR TITLE
Added stuff

### PR DESCRIPTION
**Pull request by skedastically**
Legends:

- **r:** requested icon
- **a:** `appfilter.xml` updated
- **i:** new icon provided
- **o:** alternative icon(s)
___

|App|r|a|i|o|Notes|
|-|-|-|-|-|-|
|[Vietinbank iPay](https://play.google.com/store/apps/details?id=com.vietinbank.ipay)|x||[x](https://github.com/skedastically/Arcticons/blob/main/other/vietinbank_ipay.svg)||Same logo with correct name. Perhaps delete `viettinbank.svg` and all mentions of `viettinbank` to avoid confusion/future usage.|
|[Liên Quân Mobile](https://play.google.com/store/apps/details?id=com.garena.game.kgvn)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/lien_quan_mobile.svg)||a.k.a. Arena of Valor, [icon source](https://upload.wikimedia.org/wikipedia/vi/9/93/Li%C3%AAn_quan_mobile.png)|
|Samsung Browser Lite/Go - com\.sec\.android\.app\.sbrowser\.lite|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/samsung_browser_lite.svg)|[x](https://github.com/skedastically/Arcticons/blob/main/other/samsung_browser_lite_alt.svg)|[main icon link](https://samsung-internet-lite-go.en.uptodown.com/android), [alt icon link](https://androidapksfree.com/samsung-internet-lite/com-sec-android-app-sbrowser-lite/)|
|Nothing system service - com.nothing.appservice|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/nothing_system_service.svg)|[x](https://github.com/skedastically/Arcticons/blob/main/other/nothing_system_service_alt.svg)|Please decide which is better|
|[robotfindskitten](https://f-droid.org/en/packages/info.staticfree.android.robotfindskitten/)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/robotfindskitten.svg)|[x](https://github.com/skedastically/Arcticons/blob/main/other/robotfindskitten_alt.svg)|Please decide|
|`hsbc.svg`|||[x](https://github.com/skedastically/Arcticons/blob/main/other/hsbc.svg)||Renamed from `hsbcuk.svg`|
|[HSBC UK](https://play.google.com/store/apps/details?id=uk.co.hsbc.hsbcukmobilebanking)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/hsbcuk.svg)||Added country code to `hsbc.svg`|
|[HSBC Singapore](https://play.google.com/store/apps/details?id=sg.com.hsbc.hsbcsingapore)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/hsbc_singapore.svg)||Added country code to `hsbc.svg`|
|[HSBC Vietnam](https://play.google.com/store/apps/details?id=vn.hsbc.hsbcvietnam)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/hsbc_vietnam.svg)||Added country code to `hsbc.svg`|
|[Domino's Pizza Pakistan](https://play.google.com/store/apps/details?id=pk.com.dominos)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/dominos_pk.svg)||Added country code to `dominos.svg` (important because halal)|
|[Domino's Canada](https://play.google.com/store/apps/details?id=ca.dominospizza)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/dominos_ca.svg)||Added country code to `dominos.svg`
|[Domino's Pizza México](https://play.google.com/store/apps/details?id=com.molo.dominos)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/dominos_mx.svg)||Added country code to `dominos.svg`|
|[MB Bank](https://play.google.com/store/apps/details?id=com.mbmobile)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/mb_bank.svg)|[x](https://github.com/skedastically/Arcticons/blob/main/other/mb_bank_alt.svg)||
|[MBS Mobile](https://play.google.com/store/apps/details?id=vn.com.mbs.m24np)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/mbs_mobile.svg)|[x](https://github.com/skedastically/Arcticons/blob/main/other/mbs_mobile_alt.svg)||
|[Wordament](https://play.google.com/store/apps/details?id=com.microsoft.wordament)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/wordament.svg)|[x](https://github.com/skedastically/Arcticons/blob/main/other/wordament_alt.svg)||
|[ACB ONE](https://play.google.com/store/apps/details?id=mobile.acb.com.vn)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/acb_one.svg)|||
|[Ap15](https://play.google.com/store/apps/details?id=com.aurhe.ap15)|x|x|[x]([x](https://github.com/skedastically/Arcticons/blob/main/other/ap15.svg))|||
|[B612](https://play.google.com/store/apps/details?id=com.linecorp.b612.android)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/b612.svg)|||
|[Caixa](https://play.google.com/store/apps/details?id=br.com.gabba.Caixa)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/caixa.svg)|||
|[Mint Mobile](https://play.google.com/store/apps/details?id=com.uvnv.mintsim)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/mint_mobile.svg)|||
|[Bigo Live Lite](https://play.google.com/store/apps/details?id=sg.bigo.live.lite)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/bigo_live_lite.svg)|||
|[eFootball](https://play.google.com/store/apps/details?id=jp.konami.pesam)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/efootball™.svg)
|[ESPN](https://play.google.com/store/apps/details?id=com.espn.score_center)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/espn.svg)|||
|[LG ThinQ](https://play.google.com/store/apps/details?id=com.lgeha.nuts)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/lg_thinq.svg)|||
|[NT Calculadora](https://play.google.com/store/apps/details?id=thanhletranngoc.calculator.pro)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/nt_calculadora.svg) |||
|[TodoTree](https://f-droid.org/en/packages/nl.tsmeets.todotree)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/todotree.svg)|||
|[MyT](https://play.google.com/store/apps/details?id=app.mytoyota.toyota.com.mytoyota)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/myt.svg)|||
|[Today Weather](https://play.google.com/store/apps/details?id=mobi.lockdown.weather)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/todayweather.svg)|||
|[Public](https://play.google.com/store/apps/details?id=com.public.app)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/public.svg)|||
|[OVPN](https://play.google.com/store/apps/details?id=com.ovpn.android)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/ovpn.svg)|||
|[Parsec](https://play.google.com/store/apps/details?id=tv.parsec.client)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/parsec.svg)|||
|Samsung's [Sound Assistant](https://galaxystore.samsung.com/detail/com.samsung.android.soundassistant?langCd=en)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/sound_assistant.svg)|||
|Samsung's [Sound Assistant](https://play.google.com/store/apps/details?id=com.sec.app.samsungprintservice)|x|x|[x](https://github.com/skedastically/Arcticons/blob/main/other/samsung_print_service_plugin.svg)|||
|Samsung's Activity Zone - com\.sec\.activemode||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/activity_zone.svg)|||
|Samsung's Emergency Zone - com\.sec\.emergencymode||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/emergency_zone.svg)|||
|[VNeID](https://play.google.com/store/apps/details?id=com.vnid)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/vneid.svg)|||
|[VietnamWorks](https://play.google.com/store/apps/details?id=com.vietnamworks.vietnamworks)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/vietnamworks.svg)|||
|[VNPay](https://play.google.com/store/apps/details?id=vnpay.smartacccount)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/v_vnpay.svg)|||
|[ShopeePay VN](https://play.google.com/store/apps/details?id=com.beeasy.toppay)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/shopeepay.svg)|||
|[My VNPT](https://play.google.com/store/apps/details?id=com.vnp.myvinaphone)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/my_vnpt.svg)|||
|[My MobiFone](https://play.google.com/store/apps/details?id=vms.com.vn.mymobifone)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/my_mobifone.svg)|||
|[Tiki](https://play.google.com/store/apps/details?id=vn.tiki.app.tikiandroid)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/tiki.svg)|||
|[Timo](https://play.google.com/store/apps/details?id=io.lifestyle.plus)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/timo.svg)|||
|[Bigo Live](https://play.google.com/store/apps/details?id=sg.bigo.live)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/bigo_live.svg)|||
|[Clipious](https://apt.izzysoft.de/fdroid/index/apk/com.github.lamarios.clipious)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/clipious.svg)|||
|[TCInvest](https://play.google.com/store/apps/details?id=com.fss.tcbs.mobiletrading)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/tcinvest.svg)|||
|[ACB Safekey](https://play.google.com/store/apps/details?id=vn.mk.acb.safekey)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/acb_safekey.svg)|||
|[Finhay](https://play.google.com/store/apps/details?id=vn.finhay.finhay)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/finhay.svg)|||
|[VNDIRECT](https://play.google.com/store/apps/details?id=vn.com.vndirect.stocks)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/vndirect.svg)|||
|[VNDIRECT DStock](https://play.google.com/store/apps/details?id=vn.com.vndirect.protrade)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/vndirect_dstock.svg)|||
|[Pulsar](https://play.google.com/store/apps/details?id=com.rhmsoft.pulsar)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/pulsar.svg)|||
|[PC-Covid Viet Nam](https://play.google.com/store/apps/details?id=com.mic.bluezone)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/pccovid.svg)|||
|[Public Transport Victoria - PTV](https://play.google.com/store/apps/details?id=au.gov.vic.ptv)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/ptv.svg)|||
|[Service Victoria](https://play.google.com/store/apps/details?id=au.gov.vic.service.digitalwallet.citizen)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/service_victoria.svg)|||
|[Service WA](https://play.google.com/store/apps/details?id=au.gov.wa.digital.service.mobile.servicewa.citizen)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/servicewa.svg)|||
|[VPBank NEO](https://play.google.com/store/apps/details?id=com.vnpay.vpbankonline)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/vpbank_neo.svg)|||
|[VPBank Smart OTP](https://play.google.com/store/apps/details?id=com.vpbank.vpbanksmartotp)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/vpbank_smart_otp.svg)|||
|[Vietcombank Smart OTP](https://play.google.com/store/apps/details?id=vietcombank.itcenter.vcbsmartoptv10)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/vietcombank_smart_otp.svg)|||
|[VETC](https://play.google.com/store/apps/details?id=vn.com.vetc.app)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/vetc.svg)|||
|[Busmap](https://play.google.com/store/apps/details?id=com.t7.busmap)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/busmap.svg)|||
|[OpenWRT Manager](https://apt.izzysoft.de/fdroid/index/apk/com.hg.openwrtmanager)||x|[x](https://github.com/skedastically/Arcticons/blob/main/other/openwrt_manager.svg)|||
|[Antimine](https://f-droid.org/en/packages/dev.lucanlm.antimine/)||x||||
|Samsung's Visual Voicemail - com\.samsung\.vvm.**ui.VVMActivity**||x||||
|User Manual - com\.sec\.android\.usermanual|x|x|||
|[Domino's](https://play.google.com/store/apps/details?id=com.Dominos)||x||||
|[Techcombank](https://play.google.com/store/apps/details?id=vn.com.techcombank.bb.app)||x||||
|[Shopee Poland](https://play.google.com/store/apps/details?id=com.shopee.pl)||x||||
|[Phonograph Plus](https://apt.izzysoft.de/fdroid/index/apk/player.phonograph.plus)||x||||
|[Goodtime](https://f-droid.org/packages/com.apps.adrcotfas.goodtime/)||x||||
|[MobilePay](https://play.google.com/store/apps/details?id=fi.danskebank.mobilepay)||x||||

### CODE REVIEW

**Comment by skedastically** [comment]
- [VNPay](https://play.google.com/store/apps/details?id=vnpay.smartacccount)
- [eFootball](https://play.google.com/store/apps/details?id=jp.konami.pesam)[^1]
- [ShopeePay VN](https://play.google.com/store/apps/details?id=com.beeasy.toppay)

[^1]: requests.txt

File: other/requests.txt [hunk 1]
```diff
@@ -39,7 +39,7 @@ Last requested 1677109531.0
     
 <!-- Simpl -->
 <item component="ComponentInfo{com.simpl.android/com.simpl.android.MainActivity}" drawable="simpl" />
-https://play.google.com/store/apps/details?id=com.simpl.android
+https://play.google.com/store/apps/det	ails?id=com.simpl.android
```
Comment (bsvka) [inline other/requests.txt:42]: Remove spaces

File: other/requests.txt [hunk 2]
```diff
@@ -437,7 +416,7 @@ Requested 5 times
 Last requested 1677775564.0
     
 <!-- Cache Cleaner -->
-<item component="ComponentInfo{com.github.bmx666.appcachecleaner/com.github.bmx666.appcachecleaner.ui.activity.AppCacheCleanerActivity}" drawable="cache_cleaner" />
+<item component="ComponentInfo{com.gifthub.bmx666.appcachecleaner/com.github.bmx666.appcachecleaner.ui.activity.AppCacheCleanerActivity}" drawable="cache_cleaner" />
```
Comment (bsvka) [inline other/requests.txt:419]: gifthub -> github

**Review by bsvka — CHANGES_REQUESTED** [review]

File: other/appfilter.xml [hunk 3]
```diff
@@ -2923,7 +2927,7 @@
   <item component="ComponentInfo{io.github.robertaguilera712.cisojr4droid/io.github.robertaguilera712.cisojr4droid.activities.MainActivity}" drawable="cisojr4droid"/>
 
   <!-- Citi Mobile -->
-  <item component="ComponentInfo{com.citi.citimobile/com.citi.mobile.pt3.GlobalPhoneActivity}" drawable="citi_mobile"/>
+  <item component="ComponentInfo{com.citi.ci	bile/com.citi.mobile.pt3.GlobalPhoneActivity}" drawable="citi_mobile"/>
```
Comment (bsvka) [inline other/appfilter.xml:2930]: Mistake? This will break the icon for "Citi Mobile"

Comment (skedastically) [reply other/appfilter.xml:2930]: More typos... I'll fix them

Thanks for looking out

**Comment by Donnnno** [comment]
Wow, this is a huge PR! I'll have to take some time on checking this one :-)

**Comment by Donnnno** [comment]
Fantastic work!

**Pull request merged**